test/astable_flag.jl

```diff
@@ -0,0 +1,124 @@
+module TestAsTableFlag
+
+using Test
+using DataFramesMeta
+using Statistics
+
+const ≅ = isequal
+
+@testset "@astable with just assignments" begin
+    df = DataFrame(a = 1, b = 2)
+
+    d = @rtransform df @astable begin
+        :x = 1
+
+        nothing
```

Comment:
Why return `nothing`?

Reply:
I wanted to make sure it could handle an expression that wasn't just assignments.